data/json/items/ranged.json

```diff
@@ -4194,7 +4194,38 @@
     "reload": 800,
     "loudness" : 175,
     "valid_mod_locations": [[ "accessories", 4 ],[ "barrel", 1 ],[ "bore", 1 ],[ "sights", 1 ],[ "mechanism", 4 ],[ "muzzle", 1 ],[ "sights", 1 ]]
- },{
+ },
+ {
+    "id": "m134",
+    "type": "GUN",
+    "symbol": "(",
+    "color": "dark_gray",
+    "name": "M134D-H Minigun",
+    "description": "The M134 Minigun is a 7.62x51 mm NATO, six-barreled machine gun.  It features Gatling-style rotating barrels with an electric motor.  Its' recoil is too powerful for firing unmounted, but if you could attach it to a vehicle and find enough ammo for it, it would become a devastating weapon. ",
+    "price": 5500000,
+    "material": "hardsteel",
+    "flags": ["MODE_BURST", "NEVER_JAMS", "MOUNTED_GUN"],
+    "skill": "rifle",
+    "ups_charges": 1,
+    "ammo": "308",
+    "weight": 19051,
```

Comment:
This weight is a bit light for even the base gun *without* the ammunition feed system and batteries; please add another six or eight kilos to the weight in order to account for them.